geozero/src/wkb/wkb_reader.rs

```diff
@@ -667,6 +701,12 @@ mod test {
             //OGR: MULTIPOINT ((10 -20 100),(0 -0.5 101))
         );
 
+        // SELECT 'MULTIPOINT (10 -20 100, 0 -0.5 101)'::geometry
```

Comment:
This comment looks like a copy paste error - can you update?

Reply:
Fixed - If you notice other small stuff like that, please feel free to commit directly to the branch as I allowed edits by maintainers! Alternatively the "Add suggestion" comment feature is a nice way to solve these in a more seamless way. 🙂